Common/include/Gauss_Elimination_core.hpp

```diff
@@ -0,0 +1,141 @@
+    su2double d0,d1,d2,d3,d4;
+    if (nVar<6) {
+#define BLOCK(i,j) block[(i)*nVar+(j)]
+    d0 = 1.0000000000000000 / BLOCK(0,0);
```

Comment:
The rounding error plays a role for Gauss-Elimination. So I just wanted to assure, that 1. indeed has fullest possible precision (double) and not just the (most common) float-precision (8 digits) with some random numbers behind those 8 digits.